src/renderer/features/nodeSlice.ts

```diff
@@ -119,7 +120,15 @@ window.electron.ipcRenderer.madara.onNodeLogs((event: any, data: string) => {
 });
 
 // set up listener to set isRunning to false when node stops
+// and terminate all running apps
 window.electron.ipcRenderer.madara.onNodeStop(() => {
+  const runningApps = useAppSelector(selectRunningApps);
```

Comment:
this doesn't work. useAppSelector is a react hook and so it can't be used inside the onNodeStop function. you should `getStore().getState()` instead

Reply:
i see, i will make another change then, thank you